atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/schema/indexing/IndexTest.java

```diff
@@ -66,9 +65,12 @@ public void testAddDelete() {
                     DataTable.Index2IdxTable.of(getTableFactory().getDataTable(txn));
             DataTable.Index3IdxTable index3 =
                     DataTable.Index3IdxTable.of(getTableFactory().getDataTable(txn));
-            assert index1.getRange(RangeRequest.builder().build()).count() == 1;
-            assert index2.getRange(RangeRequest.builder().build()).count() == 2;
-            assert index3.getRange(RangeRequest.builder().build()).count() == 1;
+            assertThat(index1.getRange(RangeRequest.builder().build()).count() == 1)
+                    .isTrue();
+            assertThat(index2.getRange(RangeRequest.builder().build()).count() == 2)
+                    .isTrue();
+            assertThat(index3.getRange(RangeRequest.builder().build()).count() == 1)
+                    .isTrue();
```

Comment:
```suggestion
            assertThat(index1.getRange(RangeRequest.builder().build()).count()).isEqualTo(1L);
            assertThat(index2.getRange(RangeRequest.builder().build()).count()).isEqualTo(2L);
            assertThat(index3.getRange(RangeRequest.builder().build()).count()).isEqualTo(1L);
```

Reply:
I believe the robots will update this next time they run -- it tries to make a minimal improvement each sweep.